### PR TITLE
build(docs-infra): update docs examples to Angular v10

### PR DESCRIPTION
**Pull request by gkalpak**
This commit updates the docs examples to Angular v10.1.3. In addition to updating the dependencies versions, it also updates the project's structure and config to more closely match what a new v10 CLI app would look like.

See, also, the [diff][1] between a basic v9.1.4 CLI app and a v10.1.3 one.

[1]: https://github.com/cexbrayat/angular-cli-diff/compare/9.1.4..10.1.3

### CODE REVIEW

File: aio/content/examples/http/src/app/uploader/uploader.service.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ export class UploaderService {
   // }
 
   upload(file: File) {
-    if (!file) { return; }
+    if (!file) { return of<string>(); }
```
Comment (petebacondarwin) [inline aio/content/examples/http/src/app/uploader/uploader.service.ts:27]: I guess adding a return type to this method would have made it easier to catch this error?

**Review by petebacondarwin — APPROVED** [review]
I am confident that you will follow up this PR with further ones to resolve the TODOs :-)

**Review by petebacondarwin — APPROVED** [review]
Reviewed-for: global-docs-approvers

These minor example changes don't need explicit approval from `fw-http` and `fw-testing`.

**Comment by gkalpak** [comment]
Thx for reviewing so quickly, @petebacondarwin!

> I am confident that you will follow up this PR with further ones to resolve the TODOs :-)

I would love to, but no promises. I counted more than 900 failures for the two new tslint TODOs alone. (I wouldn't dare to count the failures for the two tsconfig strictness TODOs 😛).

**Pull request closed without merging**